Source/WTF/wtf/Ref.h

```diff
@@ -276,6 +276,18 @@ inline bool is(const Ref<ArgType, PtrTraits>& source)
     return is<ExpectedType>(source.get());
 }
 
+template<typename Target, typename Source, typename PtrTraits>
+inline Target& downcast(Ref<Source, PtrTraits>& source)
+{
+    return static_cast<Target&>(source.get());
```

Comment:
You should probably be calling downcast<>() here. Otherwise, this downcast() version does not assertion checking, which defeats the purpose of downcast.